setup.py

```diff
@@ -92,8 +93,8 @@
         'stsci.imagestats',
         'stsci.skypac',
         'stsci.stimage',
-        'stwcs',
-        'tweakwcs>=0.4.0',
+        'stwcs>=1.5.0',
+        'tweakwcs>=0.4.6',
```

Comment:
why not 0.5.0?

Reply:
```
Searching for stwcs>=1.5.0
Reading https://pypi.org/simple/stwcs/
No local packages or working download links found for stwcs>=1.5.0
error: Could not find suitable distribution for Requirement.parse('stwcs>=1.5.0')
script returned exit code 1
```

Reply:
That is because stwcs stable release is only at 1.4.2. Anything after are only either alpha or rc and never pushed to PyPI.

Reply:
Regarding the tweakwcs version pin: changes made in 0.4.6 addressed problems that affected the operation of the alignment code, whereas the bug fixes that followed may make the code a little more accurate, it did not affect whether the code would run or not.

As for STWCS: there are release candidate tags on master which satisfy this requirement according to github.  Unfortunately, our code absolutely requires these newer versions.  So, they have been turned into pip installable packages yet just means that our code will have to wait to be delivered until stwcs (and photutils, for that matter) have been 'released' and turned into pip-installable products.

Reply:
For `pip install` to work properly, `stwcs` release will have to be stable, RC won't cut it unless you want to do `pip install --pre`.

Reply:
> just means that our code will have to wait to be delivered until stwcs (and photutils, for that matter) have been 'released'

@jhunkeler This complicates release of bug fixes for `drizzlepac` that users reported and are waiting for a release. Would it be possible to make a `drizzlepac` release without HLA pipeline?

Reply:
> release without HLA pipeline?

Without HLA = no. With broken HLA = yes.

Reply:
huh?

Reply:
If I understand this diff correctly, HLA is already in drizzlepac `master`. You can make a release without merging this, yes. But it does not mean releasing drizzlepac *without* HLA, technically.

Reply:
I am confident in @jhunkeler supernatural abilities.

Reply:
To release without HLA we'd need to `git revert` all of the HLA commits on `master`. 

So is it possible? Yes.
Will it look ugly? Absolutely.
Could there be complications? Probably.

Reply:
Theoretically... You could branch off the repo at an older hash before first HLA merge, then re-apply any drizzlepac-only bug fixes as backports, and tag a release from *that* branch... But yeah, Joe is correct in this Three Answers ™️ .

Reply:
Another option is to make `stwcs` an *optional* dependency. That is, everything would still install, but certain parts requiring `stwcs` would throw an error if it is not installed or too old. Joe hates this because it complicates the delivery process.

Reply:
I interpret @jhunkeler Three Answers&trade; somewhat differently:

- _"Will it look ugly? Absolutely."_ Relative to some idealized situation? Maybe... Compared to the current state? I doubt.
- _"Could there be complications? Probably."_ Hopefully not or negligible complications. 🤞

Reply:
After discussing this with Nadia, the real solution to this issue would be to establish a requirements file where this version dependency is specified for stwcs, stsci.tools, and all other critical packages, while removing the ('offending', premature) changes to the setup.py file.

Reply:
The only thing holding up the release of everything is the configuration of the package for installation and testing.  I would hope it would actually take more effort to separate out the changes for the non-HLA-related changes than it would take to fix the configuration issues.  In addition, There are additional issues raised by the testing that has been done which need to be addressed before the non-HLA-related changes can be release anyway.  As a result, I would see it as counter-productive to separate the sets of changes, and instead continue working toward releasing all the code now.